package.json

```diff
@@ -13,33 +13,33 @@
     "lint:fix": "eslint --ext .js,.vue src --fix"
   },
   "dependencies": {
+    "@nextcloud/axios": "^1.3.2",
+    "@nextcloud/initial-state": "^1.1.1",
+    "@nextcloud/router": "^1.0.2",
+    "@nextcloud/vue": "^1.4.1",
     "debounce": "^1.2.0",
-    "nextcloud-axios": "^0.2.1",
-    "nextcloud-initial-state": "0.0.3",
-    "nextcloud-router": "0.0.9",
-    "nextcloud-vue": "^0.13.0",
     "vue": "^2.6.11",
     "vue-fragment": "^1.5.1",
     "vuex": "^3.1.3"
   },
   "browserslist": [
-    "extends browserslist-config-nextcloud"
+    "extends @nextcloud/browserslist-config"
   ],
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/preset-env": "^7.9.0",
+    "@nextcloud/browserslist-config": "^1.0.0",
+    "@nextcloud/eslint-config": "^2.0.0",
```

Comment:
Missing the update of eslint config
https://github.com/nextcloud/user_retention/blob/master/.eslintrc.js